templates/show-task-table.html

```diff
@@ -5,6 +5,16 @@
     <title>Task-Table</title>
     <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.12.1/bootstrap-table.min.css">
     <script src="//cdnjs.cloudflare.com/ajax/libs/bootstrap-table/1.12.1/bootstrap-table.min.js"></script>
+    <script type="text/javascript">
+    $(document).ready(function(){
+        tasks = {{tasks|safe}};
+        task_names = [];
+        $.each(tasks, function(index, value){
+	        task_names.push(value.article_name);
+	    });
+        awesomplete.list = task_names;
```

Comment:
Does something like:

```javascript
awesomplete.list = tasks.map(t => t.article_name);
```

work? Personal preference, but I'd say that's both clearer and more concise.

Reply:
Agreed, it's a cleaner way. Thanks.